ts/node/sql.ts

```diff
@@ -972,11 +971,13 @@ function removeAllMessagesInConversation(
   if (!conversationId) {
     return;
   }
+  const inst = assertGlobalInstanceOrInstance(instance);
 
-  // Our node interface doesn't seem to allow you to replace one single ? with an array
-  assertGlobalInstanceOrInstance(instance)
-    .prepare(`DELETE FROM ${MESSAGES_TABLE} WHERE conversationId = $conversationId`)
-    .run({ conversationId });
+  inst.transaction(() => {
```

Comment:
what is the benefit of wrapping this prepare statement in a transaction?

Reply:
This was a test to run the triggers called while deleting messages only once, in an attempt to batch them at the database level. But I don't think it helps, what helped was the change of id to rowId in the fts virtual table

Reply:
👍